[Map Change] Armeria (Cyberiad) by KANS

Spircen opened the pull request:
## What Does This PR Do
Agrega la armería diseñada por Kans, tiene varios problemas para ocupar mapmerge asi que yo me encargo de ayudarle a hacer el PR de su nuevo mapa. Amplia el area y la unifica a la par que agrega mas hardsuits de seguridad a su departamento, armamento, munición y armaduras. A la par repara unas cuantas cámaras que se me pasaron en un mapeo anterior. Y agrega un bombillo a cargo. Chi Cheñol

## Why It's Good For The Game
Ampliar la armería es una bonifican tanto para motivar a mas personal a jugar en seguridad como para que el departamento no dependa únicamente de pedir mas armamento, actualmente la armería se encontraba demasiado vacía para su espacio total, con un lugar mas amplio y reforzado para que se sienta la importancia del lugar. Y lo que Kans diga... si eso. 

## Images of changes

                                          Nueva Armeria (Linea Rosa SecBot)
![image](https://user-images.githubusercontent.com/46639834/77984840-f99f8800-72cf-11ea-81ec-f5b64db4dad1.png)

                                            Version Actual
![image](https://user-images.githubusercontent.com/46639834/79057828-59c8ef00-7c2b-11ea-98bb-76995c7c045b.png)

                                             Dream Maker
![image](https://user-images.githubusercontent.com/46639834/79057835-64838400-7c2b-11ea-8e13-0bce1aae9a44.png)



## Changelog
:cl:
add: Armería diseñada por Kans
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Spircen committed:
Perdi mi ojo de venado, soy un ser de oscuridad. Perdi mi vida en un rosario entre milagros de laton

N0thy commented:
Solo me quejaría del montón de grilles que ahí arribita haha.

Spircen commented:
Existen en teoría para que si alguien intenta entrar por el espacio el warden escuche todo el ruido de los grilles al ser cortados y lo mas importante suavizar impacto de meteoritos.

Magmabros commented:
Eso ya estaba solo lo moví  de lugar nothin y repare algunos que quite tal vez por eso se ve mas grueso
(Soy Kans)

Sauronato commented:
Doble pared reforzada para entrar en las armas desde el espacio, pobres nukes xD

Magmabros commented:
De todos modos solo es de ese lado si avanzas te puedes conseguir un estear egg en el espacio

Tilk1 commented:
Una curiosa reforma. Algunos puntos que vi:
- Esto vuelve mucho mas dificil robar la armeria y algo molesto. ¿Actualmente hay muchos casos de robos de armeria? Veo una Doble pared reforzada y un beepsky con mucho mayor margen de movimiento  al cual no puedes acceder facilmente porque esta protegido con las windoors. La idea de que la armeria sea un punto proteger pero al mismo tiempo algo "vulnerable" hace al trabajo de warden mas entretenido. Si la armeria es un sitio tan dificil de acceder que ningun antagonista va perder el tiempo intentándolo entonces que gracia tiene el job de warden?
- Al abrir la armeria sera casi inevitable que los oficiales roben el secway y cosas que no deberian. Como las granadas barrer
- Si se llega a hacer una brecha en la parte del armamento no hay forma deq se recupere el aire. Y es muy poco 1 vent y 1 scrubber para un espacio tan enorme.
- Soy partidario de quitar el WT-550 Autorifle.   (Yase que actualmente se encuentra)  Pero seguridad por cuestiones de balance y por lore no deberia tener acceso a armamento balistico de este tipo. Las armas laseres van mas con la cultura espacial de NT  y las balas, balisticas con el Syndicate. Estaba bien cuando se tenia que pedir por cargo, ahora es el primer objeto que lootean cuando abren la armeria por su poder de fuego

Spircen commented:
Todo lo contrario, actualmente un ingeniero únicamente debe hackear una puerta, matar un secborg y desarmar una ventana para conseguir un arma. Anteriormente necesitaba hackear dos puertas para poder llegar hasta un arma. 

Tiene sentido que un oficial tome el secway en un caso donde ya se vio la necesidad de romper en la armeria, todo debe estar bastante jodido para romper las reglas. 

Si, agregare otro scrubber y un vent. 

No entiendo eso de partidario de X, no veo mas gente  queriendo eliminar el autorifle de seguridad, los autorifles son un arma característica de NT, no por nada ciencias tiene 5 tipos distinto de balas para fabricar, únicamente diseñados para este modelo propietario de NT.

Spircen committed:
Merge branch 'master' into kans

Spircen committed:
Merge remote-tracking branch 'upstream/master' into kans

Spircen committed:
Merge branch 'master' into kans

Spircen committed:
New Instead of Initialize

-Esto evita que nos atasque de Runtimes DD al iniciar el server, hasta que Paradise de una nueva forma para resolver los runtimes que genera.

Spircen committed:
Armeria de Kans y FOCO

Spircen committed:
Merge branch 'master' into kans

Spircen committed:
Revert "Merge branch 'master' into kans"

This reverts commit 0cea57ca82aaeeb022f8c2675841a70f50c3a327, reversing
changes made to 5708068e76ef8f41884904cf0ee0b18039458fce.

Spircen committed:
Merge remote-tracking branch 'upstream/master' into kans

Spircen committed:
Cambios Finales a la armeria

-Elimina cables que llevan a nada debajo de Holopads
-Disminuye la cantidad de beanbags y auto rifle ammo

Spircen commented:
Vale, ya quedaron resueltos todos los balances de la municion, las camaras y esto de paso resuelve varios Step_X que se pasaron en el PR de chacatrunco
